airbyte-integrations/connectors/source-xero/manifest.yaml

```diff
@@ -1692,55 +1691,91 @@ spec:
-                examples:
-                  - Client_Secret
-                airbyte_secret: true
-          - type: object
-            title: Bearer Access Token
-            required:
-              - access_token
-              - auth_type
-            properties:
-              auth_type:
-                type: string
-                const: oauth2_access_token
-                order: 0
-                title: Authentication Method
-                examples:
-                  - access_token
-              access_token:
-                type: string
-                description: The access token used to call the Xero API.
-                title: Access Token
-                airbyte_secret: true
+      client_id:
+        type: string
+        description: Your Xero application's Client ID.
+        title: Client ID
+        examples:
+          - Client_ID
         order: 2
-        title: Authentication Method
+      client_secret:
+        type: string
+        description: Your Xero application's Client Secret.
+        title: Client Secret
+        examples:
+          - Client_Secret
+        airbyte_secret: true
+        order: 3
+      client_access_token:
+        type: string
+        title: Access token
+        airbyte_secret: true
+        order: 4
+      client_refresh_token:
+        type: string
+        title: Refresh token
+        airbyte_secret: true
+        order: 5
     additionalProperties: true
+  advanced_auth:
+    auth_flow_type: oauth2.0
+    oauth_config_specification:
+      oauth_connector_input_specification:
+        consent_url: https://login.xero.com/identity/connect/authorize?response_type=code&client_id={{ client_id_value }}&scope={{ scope_value }}&redirect_uri={{ redirect_uri_value }}
+        scope: offline_access accounting.transactions.read accounting.reports.read accounting.budgets.read accounting.reports.tenninetynine.read accounting.journals.read accounting.settings.read accounting.contacts.read accounting.attachments.read assets.read files.read projects.read openid
+        access_token_url: https://identity.xero.com/connect/token
+        access_token_headers:
+          Authorization: "{{ (client_id_value ~ ':' ~ client_secret_value) | b64encode }}"
+        access_token_params:
+          grant_type: authorization_code
+          code: "{{ auth_code_value }}"
+          redirect_uri: "{{ redirect_uri_value }}"
+      complete_oauth_output_specification:
+        required:
+          - access_token
+          - refresh_token
+          - token_expiry_date
+        properties:
+          access_token:
+            type: string
+            path_in_connector_config:
+              - client_access_token
+            path_in_oauth_response:
+              - access_token
+          refresh_token:
+            type: string
+            path_in_connector_config:
+              - refresh_token
+            path_in_oauth_response:
+              - refresh_token
+          token_expiry_date:
```

Comment:
This has been used in the `OAuthAuthenticator`, is this intentional that we use only the `refresh_token` for the `TokenUpdater` and not the `token_expiry_date`?